spyder/plugins/editor/utils/switcher_manager.py

```diff
@@ -78,13 +78,18 @@ def create_editor_switcher(self):
             _('Start typing the name of an open file'))
 
         editorstack = self._editorstack()
+        # As editor open files are inserted at the position 0
+        # the list needs to be reversed so they show in order
```

Comment:
```suggestion

        # Since editor open files are inserted at position 0, the
        # list needs to be reversed so they're shown in order.
```

Some small improvements to make this comment easier to understand.